packages/coil-extension/src/background/services/AuthService.ts

```diff
@@ -63,7 +63,9 @@ export class AuthService extends EventEmitter {
     const resp = await this.client.whoAmI(token)
     this.log('updateWhoAmi resp', resp.data)
     if (resp.data?.whoami) {
-      this.store.user = resp.data.whoami
+      this.store.user = {
+        ...resp.data.whoami
```

Comment:
Does this do anything? It's not a deep copy, so you still referencing the gizzards of the object